README.md

```diff
@@ -2,50 +2,19 @@
 
 [![Downloads per month](https://img.shields.io/npm/dm/github-api.svg?maxAge=2592000)][npm-package]
 [![Latest version](https://img.shields.io/npm/v/github-api.svg?maxAge=3600)][npm-package]
-[![Gitter](https://img.shields.io/gitter/room/michael/github.js.svg?maxAge=2592000)][gitter]
-[![Travis](https://img.shields.io/travis/michael/github.svg?maxAge=60)][travis-ci]
-<!-- [![Codecov](https://img.shields.io/codecov/c/github/michael/github.svg?maxAge=2592000)][codecov] -->
+[![Gitter](https://img.shields.io/gitter/room/github-tools/github.js.svg?maxAge=2592000)][gitter]
+[![Travis](https://img.shields.io/travis/github-tools/github.svg?maxAge=60)][travis-ci]
+[![Codecov](https://img.shields.io/codecov/c/github/github-tools/github.svg?maxAge=2592000)][codecov]
 
-Github.js provides a minimal higher-level wrapper around Github's API. It was concieved in the context of
-[Prose][prose], a content editor for GitHub.
+Github.js provides a minimal higher-level wrapper around Github's API.
 
-## [Read the docs][docs]
-
-## Installation
-Github.js is available from `npm` or [unpkg][unpkg].
-
-```shell
-npm install github-api
-```
-
-```html
-<!-- just github-api source (5.3kb) -->
-<script src="https://unpkg.com/github-api/dist/GitHub.min.js"></script>
-
-<!-- standalone (20.3kb) -->
-<script src="https://unpkg.com/github-api/dist/GitHub.bundle.min.js"></script>
-```
-
-## Compatibility
-Github.js is tested on Node:
-* 6.x
-* 5.x
-* 4.x
-* 0.12
-
-## GitHub Tools
-
-The team behind Github.js has created a whole organization, called [GitHub Tools](https://github.com/github-tools),
-dedicated to GitHub and its API. In the near future this repository could be moved under the GitHub Tools organization
-as well. In the meantime, we recommend you to take a look at other projects of the organization.
-
-## Samples
```

Comment:
It's not removed, it's rearranged to fit https://github.com/noffle/art-of-readme